Readme.md

```diff
@@ -15,6 +15,6 @@ This project contains a starter setup for a TDD driven bowling game kata. There
 1. A player rolls 20 times and knock nothing, the expected score would be zero.
 2. A player rolls 20 balls and knock 2 pins each rolling, the expected score would be 40.
 3. A player rolls a Spare in the second frame. Calculate the score after the 3rd frame.
-4. A player gets a strike in frame 4. Calculate the score after the 4th frame.
+4. A player gets a strike in frame 4. Calculate the score after the 3rd frame.
```

Comment:
5th?

Reply:
I was just about to close it, because I thought I'd marked the wrong one.